Apps/WebClient/src/WebClient.csproj

```diff
@@ -76,7 +76,7 @@
     <!-- Include the newly-built files in the publish output -->
     <ItemGroup>
       <DistFiles Include="$(SpaRoot)dist\**" />
-      <ResolvedFileToPublish Include="@(DistFiles->'%(FullPath)')" Exclude="@(ResolvedFileToPublish)">
+      <ResolvedFileToPublish Include="@(DistFiles-&gt;'%(FullPath)')" Exclude="@(ResolvedFileToPublish)">
```

Comment:
Should this be `gt;` or `>`?  I have seen pull requests changing this from one way to the other and back again multiple times.

Reply:
It’s Mac vs PC, I try to keep it out but it goes in periodically. Doesn’t hurt anything just flips depending in what editor is used.